src/frontend/typer/eøs-trekk-i-løpende-ytelse.ts

```diff
@@ -0,0 +1,20 @@
+import type { FamilieIsoDate } from '../utils/kalender';
+
+export interface IRestTrekkILøpendeUtbetaling {
+    id: number;
+    fom: FamilieIsoDate;
+    tom: FamilieIsoDate;
+    feilutbetaltBeløp: number;
+}
+
+export interface IRestNyFeilutbetaltValutaPeriode {
+    fom: FamilieIsoDate;
+    tom: FamilieIsoDate;
+    feilutbetaltBeløp: number;
+}
+
+export interface IFeilutbetaltValutaSkjemaFelter {
+    fom: FamilieIsoDate;
+    tom: FamilieIsoDate;
+    feilutbetaltBeløp: string;
```

Comment:
Hvorfor må denne være string?

Reply:
Oppsummering fra diskusjon med Halvor: 

Når man skal sende informasjonen fra skjema til backend trenger man å vite at ingen av feltene er undefined, og at de er gyldige. Dette sjekkes ved å bruke `kanSendeSkjema()` (som validerer alle feltene), men dette er ikke nok for typescript som ville ha eksplisitt sjekk av at alle felter ikke var undefined. Siden vi vet at feltene er fyldige etter valideringen er det kun nødvendig med en sånn sjekk for å gjøre typescript fornøyd. For fom og tom ble dette løst ved å bruke en tom streng og si at den ikke kan være undefined. For beløp kunne det vært løst ved å si at default = 0, men det gjør at det kommer en 0 i input fra start som ikke fjernes når man begynner å skrive (man skriver da feks inn 0100 som blir tolket som 100). Siden input med type number returnerer en string så gikk vi for en tom streng som valideres om det er et tall før det sendes videre.